ArduCopter/events.cpp

```diff
@@ -43,33 +43,33 @@ void Copter::failsafe_radio_on_event()
     // Conditions to deviate from FS_THR_ENABLE selection and send specific GCS warning
     if (should_disarm_on_failsafe()) {
         // should immediately disarm when we're on the ground
-        gcs().send_text(MAV_SEVERITY_WARNING, "Radio Failsafe - Disarming");
+        announce_failsafe("Radio Failsafe", "Disarming");
```

Comment:
we could potentially even factor out the word " Failsafe"

Reply:
agree with this ! That would help those that love to parse status_txt to get infos ! Failsafe - System - Reason

Reply:
would be great to extend to other vehicle to unify too

Reply:
> we could potentially even factor out the word " Failsafe"

Done

Reply:
don't want to put it first ?

Reply:
gcs().send_text(MAV_SEVERITY_WARNING, "Failsafe %s - %s", type, action_undertaken);

Reply:
I'd prefer to keep this a NFC as far as the user's concerned.

It's also a bit more natural to say "Radio Failsafe" than "Failsafe Radio" - adjective/noun weirdness otherwise.